spring-cloud-dataflow-shell/src/main/java/org/springframework/cloud/dataflow/shell/command/RuntimeCommands.java

```diff
@@ -57,9 +57,13 @@
 public class RuntimeCommands implements CommandMarker {
 
 	private static final String LIST_MODULES = "runtime modules";
+	
+	private DataFlowShell dataFlowShell;
 
 	@Autowired
-	private DataFlowShell dataFlowShell;
+	public void setDataFlowShell(DataFlowShell dataFlowShell) {
```

Comment:
ultimately I think we should avoid scanning, but that's part of a bigger effort

I'm just never a fan of seeing a change that's made purely for the tests